README.md

```diff
@@ -347,9 +378,77 @@ knows Staltz is part of the group because all `group/add-member` messages have
 Staltz can see Arj has invited Mix because he's replicating Arj's
 "group/additions" feed, so Staltz starts replicating Mix's group feed.
 
+## 5. Security Considerations
+
+### 5.1. Degraded confidentiality at scale
+
+As noted in the [private-groups-original-notes], SSB Private Groups are based on
+symmetric encryption and static keys (no key rotation), which means it cannot
+guarantee [perfect-forward-secrecy] neither [post-compromise-security]. This
+makes SSB Private Groups less likely to guarantee confidentiality at scale, when
+a group has dozens or hundreds of members. If any of the members leaks the group
+secret, all past communication in the group is compromised.
+
+This also means that any group member can add new members to the group. They may
+even do so without publishing a `group/add-member` message, which effectively
+allows any group member to invite a third party that will remain undetected by
+existing group members.
+
+SSB Private Groups with several members should thus not be considered
+confidential nor forward secret, and should be used with caution. Our security
```

Comment:
I think "not confidential" is a bit rough.

> Confidential - Done or communicated in confidence; secret.

Can we guarentee secrecy? No. No encryption system guarentees that to my knowledge - there is always a way to leak. You might argue can you leak but be plausably deniable? here no, because of the signatures of messages, don't know about other things like Signal? Signal is FOSS, could I make a hacked version which leaked? probably? You'd just have to never forget the ratchet keys?

Reply:
I removed the "not confidential" and simplified this paragraph.